js/ui/grid_core/ui.grid_core.columns_resizing_reordering.js

```diff
@@ -227,6 +227,7 @@ var ColumnsSeparatorView = SeparatorView.inherit({
         if($element && (this._isShown || force)) {
             if(this._isTransparent) {
                 $element.addClass(columnsSeparatorTransparent);
+                $element.css("left", "");
```

Comment:
Why need reset the left attribute?

Reply:
Because browser shows horizontal scroll if left attribute is not reseted